src/acceptance-test/java/org/zalando/nakadi/webservice/timelines/TimelineConsumptionTest.java

```diff
@@ -18,14 +19,12 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.zalando.nakadi.domain.EventType;
-import org.zalando.nakadi.webservice.BaseAT;
-import static com.jayway.restassured.RestAssured.given;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.createEventType;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.createTimeline;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.deleteTimeline;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.publishEvent;
 
-public class TimelineConsumptionTest extends BaseAT {
+public class TimelineConsumptionTest {
```

Comment:
Well, evidently it is. I guess my question is 'why does it not extend BaseAT anymore?'